app/services/fed_email_domains.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+class FedEmailDomains
+  FED_EMAIL_DOMAINS_PATH = Rails.root.join(IdentityConfig.store.fed_domain_file_path).freeze
```

Comment:
this seems like something I would not make configurable? a file like this would need to be downloaded proactively so we'd need to know the location ahead of time?